eidpdata/src/main/java/uk/ac/ucl/eidp/data/NodeQualifier.java

```diff
@@ -1,38 +1,27 @@
-/*
- * Copyright 2015 David Guzman <d.guzman at ucl.ac.uk>.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
 package uk.ac.ucl.eidp.data;
 
 import static java.lang.annotation.ElementType.FIELD;
 import static java.lang.annotation.ElementType.METHOD;
 import static java.lang.annotation.ElementType.PARAMETER;
 import static java.lang.annotation.ElementType.TYPE;
-import java.lang.annotation.Retention;
 import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Retention;
 import java.lang.annotation.Target;
 import javax.inject.Qualifier;
 
 /**
  *
- * @author David Guzman <d.guzman at ucl.ac.uk>
+ * @author David Guzman {@literal d.guzman at ucl.ac.uk}
  */
 @Qualifier
 @Retention(RUNTIME)
 @Target( {TYPE, METHOD, FIELD, PARAMETER} )
 public @interface NodeQualifier {
-    
-    NodeType value();
+  
+  /**
```

Comment:
\[Checkstyle\] WARNING: First sentence of Javadoc is incomplete \(period is missing\) or not present\.